lib/drivers/docker/index.js

```diff
@@ -0,0 +1,294 @@
+'use strict';
+
+var _ = require('lodash');
+var async = require('async');
+var bl = require('bl');
+var debug = require('debug')('strong-pm:docker:driver');
+var Docker = require('dockerode');
+var EventEmitter = require('events').EventEmitter;
+var Image = require('./image');
+var inherits = require('util').inherits;
+var isError = require('util').isError;
+
+module.exports = exports = DockerDriver;
+
+function DockerDriver(opts) {
+  if (!opts.baseDir || !opts.console || !opts.server) {
+    throw Error('Requires baseDir, console, and server');
```

Comment:
if you hit this, there's no way to know what was missing, did you see `lib/util.js:mandatory()`?